### ISSUE TITLE
Running code for classification tasks with more than 2 labels.

### ISSUE THREAD

**Issue by ashim95** [LilyNLP/ADFAR#4]
Hi @LilyNLP ,

I am trying to run your model for `AGNews` dataset which has 4 classes. I have made some changes in the data processors to accommodate for this, but the model is not training properly, which I suppose is because of some modeling issue (`BertForSequenceClassificationAdvV2` or `BertForSequenceClassificationAdvV3`). I have also changed the codes in `run_simplification.py`, `get_pure_adversaries`, and `combine_data.py`  to accommodate for change in number of labels. 

I am using the following command to run the model:
```
python run_classification_adv.py --task_name ag_news-adv --max_seq_len 512 --do_train  --do_eval --attention 2  \
--data_dir data/ag_news/combined_data/4times_adv_0-9/ --output_dir experiments/ag_news/4times_adv_double_0-9 \
--model_name_or_path bert-base-uncased --per_device_train_batch_size 16 --per_device_eval_batch_size 16 --learning_rate 3e-5 \
--num_train_epochs 5 --svd_reserve_size 0 --evaluation_strategy epoch
``` 

My question is: If I want to run your model for classification tasks with more than 2 labels, how can I do that? Can you please help with this issue? 

Thank you,
